runtime/cudaq/qis/managers/BasicExecutionManager.h

```diff
@@ -224,31 +202,29 @@ class BasicExecutionManager : public cudaq::ExecutionManager {
       mutable_controls.push_back(e);
 
     std::vector<cudaq::QuditInfo> mutable_targets;
-    for (auto &t : targets) {
+    for (auto &t : targets)
       mutable_targets.push_back(t);
-    }
 
     if (isAdjoint || !adjointQueueStack.empty()) {
-      for (std::size_t i = 0; i < params.size(); i++) {
+      for (std::size_t i = 0; i < params.size(); i++)
         mutable_params[i] = -1.0 * params[i];
-      }
-      if (mutable_name == "t") {
+      if (mutable_name == "t")
```

Comment:
```suggestion
      if (gateName == "t")
```
to avoid making a copy in all cases (line 212) when the next couple of cases overwrite the copy anyway.